h2o-py/h2o/estimators/gbm.py

```diff
@@ -58,6 +58,20 @@ def training_frame(self):
         Id of the training data frame.
 
         Type: ``H2OFrame``.
+
```

Comment:
@angela0xdata @hannah-tillman this is how it looks like once the code is generated